packages/react-native/ReactCommon/react/renderer/components/textinput/BaseTextInputProps.h

```diff
@@ -72,6 +72,8 @@ class BaseTextInputProps : public ViewProps, public BaseTextProps {
 
   bool multiline{false};
 
+  bool disableKeyboardShortcuts{};
```

Comment:
nit: it's better to explicitly set the default value

Reply:
Done - 5041880c6ca2264ef3362bdcd5e4b23be6403fcd.